sebs/benchmark.py

```diff
@@ -430,11 +430,10 @@ def install_dependencies(self, output_dir):
             )
         else:
             repo_name = self._system_config.docker_repository()
-            image_name = "build.{deployment}.{language}.{runtime}-{version}".format(
+            image_name = "build.{deployment}.{language}.{runtime}".format(
```

Comment:
We added version tags recently, but I didn't manage to update the images in DockerHub. Please remove this change, and I will update images with new tags :)

Reply:
Sure!

However, would it be useful if the benchmark suite could fall back to images w/o version tags when images w/ version cannot be found on DockerHub? If so, I could implement it here.